tests/test_decay_angles_dialog.py

```diff
@@ -45,6 +46,71 @@ def test_open_and_close_decay_angles_dialog(cpt_widget, qtbot: QtBot, click_twic
     assert not cpt_widget.decay_angles_dlg.cal_layer.visible
 
 
+def test_decay_vertex_update(cpt_widget):
+
+    cpt_widget.particle_decays_menu.setCurrentIndex(4)
+    dialog = cpt_widget._on_click_decay_angles()
+
+    # Move the decay vertex of the Lambda
+    data = dialog.cal_layer.data
+    old_value = data[0][0][0]
+    data[0][0][0] = old_value + 50
+    dialog.cal_layer.data = data  # For some reason, this doesn't trigger the change event
+    dialog.cal_layer.events.data(action="changed", data_indices=(0,))
+
+    # Check that the decay vertex has been updated
+    assert dialog.cal_layer.data[0][0][0] == old_value + 50
+    assert (dialog.cal_layer.data[1][0] == data[0][0]).all()
+    assert (dialog.cal_layer.data[2][0] == data[0][0]).all()
```

Comment:
```suggestion
    assert dialog.cal_layer.data[0][0][0] == old_value + 50, "The decay vertex has not been updated!"
    assert (dialog.cal_layer.data[1][0] == data[0][0]).all()
    assert (dialog.cal_layer.data[2][0] == data[0][0]).all()
```

Can you add explanatory strings to the other guys too? I don't get what condition they're testing.